pkgs/development/python-modules/pre-commit-hook-ensure-sops/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, applyPatches, fetchFromGitHub, ruamel-yaml }:
+
+buildPythonPackage rec {
+  pname = "pre-commit-hook-ensure-sops";
+  version = "1.1";
```

Set the format explicitly.

```suggestion
  pname = "pre-commit-hook-ensure-sops";
  version = "1.1";
  format = "setuptools";
```